i18n/config.json

```diff
@@ -119,6 +119,16 @@
     "enabled": false,
     "default": false
   },
+  {
+    "code": "pt",
+    "localName": "Português",
+    "name": "Português",
+    "langDir": "ltr",
+    "dateFormat": "MM.DD.YYYY",
+    "hrefLang": "pt",
+    "enabled": true,
+    "default": true
```

Comment:
This is wrong, Portuguese should not be the default language

Reply:
@ovflowd, it is resolved now.